R/ospsuite.utils-env.R

```diff
@@ -0,0 +1,45 @@
+# Environment that holds various global variables and settings for the ospsuite.utils,
+# It is not exported and should not be directly manipulated by other packages.
+ospsuiteUtilsEnv <- new.env(parent = emptyenv())
+
+# name of the package. This will be used to retrieve information on the package at run time
+ospsuiteUtilsEnv$packageName <- "ospsuite.utils"
+
+ospsuiteUtilsEnv$suiteName <- "Open Systems Pharmacology"
+
+# Separator defined in OSPSuite.Core.
+ospsuiteUtilsEnv$pathSeparator <- "|"
+
+# Default values for the formatNumerics() helper function
+ospsuiteUtilsEnv$formatNumericsDigits <- 5
```

Comment:
This was 5 in ospsuite, but the magic number in `formatNumerics` here was 2... Dunno whats better.